WooCommerce/Classes/ViewRelated/Upgrades/OwnerUpgradesView.swift

```diff
@@ -1,75 +1,111 @@
 import SwiftUI
+import Yosemite
+import WooFoundation
 
 struct OwnerUpgradesView: View {
-    @State var upgradePlan: WooWPComPlan
-    @State var isPurchasing = false
-    let purchasePlanAction: () -> Void
-    @State var isLoading: Bool = false
+    @State var upgradePlans: [WooWPComPlan]
+    @State var isPurchasing: Bool
+    let purchasePlanAction: (WooWPComPlan) -> Void
+    @State var isLoading: Bool
```

Comment:
The properties marked with underscore went a bit over my head here. Is this the same as providing initial values as follows?

```suggestion
    @State var upgradePlans: [WooWPComPlan] = []
    @State var isPurchasing: Bool = false
    let purchasePlanAction: (WooWPComPlan) -> Void
    @State var isLoading: Bool = false
```

Reply:
Yeah, it's just that when we do it in an init, we need to make the `State` with an initialValue. With the state values being private, we don't have a free init method for them, so this becomes required if we want to be able to set them when we make the view (which we do)